ui/README.md

```diff
@@ -0,0 +1,44 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Good to a line or two specific to our project.

Reply:
done.